spec/cb/clients/job_spec.rb

```diff
@@ -12,108 +12,6 @@
 
 module Cb
   describe Cb::Clients::Job do
-    describe '#search' do
```

Comment:
@oochelz beat me to it. I'm not seeing the client be thoroughly tested after deleting all of these or maybe am missing something.